src/website/views/components/tabs.pug

```diff
@@ -15,208 +15,445 @@ p.-text
+    .-p--3(style="background-color:#eee;")
+      .-bg--white.-p--3
+        ul.a-tabs.-vertical
           li.-active
-            a(href='#') Subtab Link
+            a(href='#') Active tab
+            ul.a-tabs__subtabs
+              li.-active
+                a(href='#') Subtab Link
+              li
+                a(href='#') Subtab Link
+              li
+                a(href='#') Subtab Link
           li
-            a(href='#') Subtab Link
+            a(href='#') Tab link
           li
-            a(href='#') Subtab Link
-      li
-        a(href='#') Tab link
-      li
-        a(href='#') Tab link
-  :code(lang='html')
-    <ul class="a-tabs -vertical">
-      <li class="-active">
-        <a href="#">Active tab</a>
-        <ul class="a-tabs__subtabs">
-          <li class="-active">
-            <a href="#">Subtab Link</a>
-          </li>
-          <li>
-            <a href="#">Subtab Link</a>
-          </li>
-          <li>
-            <a href="#">Subtab Link</a>
-          </li>
-        </ul>
-      </li>
-      <li>
-        <a href="#">Tab link</a>
-      </li>
-      <li>
-        <a href="#">Tab link</a>
-      </li>
-    </ul>
+            a(href='#') Tab link
+    :code(lang='html')
+      <ul class="a-tabs -vertical">
+        <li class="-active">
+          <a href="#">Active tab</a>
+          <ul class="a-tabs__subtabs">
+            <li class="-active">
+              <a href="#">Subtab Link</a>
+            </li>
+            <li>
+              <a href="#">Subtab Link</a>
+            </li>
+            <li>
+              <a href="#">Subtab Link</a>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="#">Tab link</a>
+        </li>
+        <li>
+          <a href="#">Tab link</a>
+        </li>
+      </ul>
+  #a-tabs-panel--subtabs-inverse.a-tabs-panel
+    .-p--3(style="background-color:#eee;")
+      .-bg--black.-p--3
+        ul.a-tabs.-inverse.-vertical
+          li.-active
+            a(href='#') Active tab
+            ul.a-tabs__subtabs
+              li.-active
+                a(href='#') Subtab Link
+              li
+                a(href='#') Subtab Link
+              li
+                a(href='#') Subtab Link
+          li
+            a(href='#') Tab link
+          li
+            a(href='#') Tab link
+    :code(lang='html')
+      <ul class="a-tabs -inverse -vertical">
+        <li class="-active">
+          <a href="#">Active tab</a>
+          <ul class="a-tabs__subtabs">
+            <li class="-active">
+              <a href="#">Subtab Link</a>
+            </li>
+            <li>
+              <a href="#">Subtab Link</a>
+            </li>
+            <li>
+              <a href="#">Subtab Link</a>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="#">Tab link</a>
+        </li>
+        <li>
+          <a href="#">Tab link</a>
+        </li>
+      </ul>
 
 h3 Bordered
```

Comment:
@jllr missing support for -inverse. let's set the bottom border color to grey 80 for inverse tabs